tests/conftest.py

```diff
@@ -46,11 +46,19 @@ def pytest_configure(config: Config) -> None:
 
 
 def pytest_collection_modifyitems(config: Config, items: list[Function]) -> None:
+    if config.getoption("--release") and config.getoption("--runslow"):
+        raise ValueError("You cannot run the release tests and slow tests simultaneously.")
     skip_release = pytest.mark.skip(reason="need --release to run")
+    skip_non_release = pytest.mark.skip(reason="only running release tests")
     if not config.getoption("--release"):
         for item in items:
-            if "release" in item.keywords:
+            parametrized_test_name = [x for x in item.keywords][0]
+            if "release" in item.keywords and "test_slow_tests" not in parametrized_test_name:
                 item.add_marker(skip_release)
+    else:
+        for item in items:
+            if "release" not in item.keywords:
+                item.add_marker(skip_non_release)
```

Comment:
Wouldn't something like this be cleaner:
```python
for item in items:
    if "release" not in item.keywords:
        item.add_marker(skip_non_release)
    elif "test_slow_tests" not in [x for x in item.keywords][0]:
        item.add_marker(skip_release)
```

Reply:
We need to distinguish between running with the release flag and running without it. This approach would mean only the slow tests in release get run with --runslow.